README.rst

```diff
@@ -125,7 +125,7 @@ To enable all algorithms at once, do the following:
 
 ``conda install pygmo``
 
-``pip install fides>=0.7.1`` (Make sure you have at least 0.7.1)
+``pip install fides>=0.7.4``
```

Comment:
Please test if it works with `0.7.1` as well now and put a comment for the minimum required version back in (also in other places)

Reply:
Done.